pkgs/development/python-modules/whool/default.nix

```
@@ -0,0 +1,55 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, git
+, hatch-vcs
+, importlib-metadata
+, lib
+, manifestoo-core
+, pytestCheckHook
+, pythonOlder
+, tomli
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "whool";
+  version = "1.0.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "sbidoul";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-skJoMDIgZgRjfp4tsc6TKYVe09XBvg8Fk2BQfqneCYI=";
+  };
+
+  build-system = [ hatch-vcs ];
+
+  dependencies = [
+    manifestoo-core
+    wheel
+  ] ++ lib.optionals (pythonOlder "3.11") [
+    tomli
+  ] ++ lib.optionals (pythonOlder "3.8") [
+    importlib-metadata
+  ];
```

```suggestion
  ];
```

We don't have such version now

---

However, given there are downstream projects such as https://github.com/cachix/nixpkgs-python/ that still support those, don't you think it's good to have this specification here? Just like upstream has those markers in their pyproject.toml.

FWIW in the Odoo world, it's common to support deployments for more time than one would expect...

---

You should support it downstream if necessary.
Python 3.7 is already EOL.